collectors/lib/samza_custom_metric_reporter.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+from samza_metric_reporter import SamzaMetricReporter
+
+class SamzaCustomMetricReporter(SamzaMetricReporter):
+
+    """
+    This reporter only prints custom samza jobs related metrics. All
+    generic Samza metrics, such as number of messages processed,
+    are reported by SamzaMetricReporter.
+    """
+
+    def __init__(self, consumer_group_id, kafka_bootstrap_servers, kafka_metrics_topic='samza_metrics'):
+        SamzaMetricReporter.__init__(self, consumer_group_id, kafka_bootstrap_servers, kafka_metrics_topic)
+        self.methods_to_run = [self.report_samza_custom_metrics]
+
+    def report_samza_custom_metrics(self, metrics_raw, header_raw):
+        class_to_metric_name_map = {
```

Comment:
I didn't pass this information as a parameter, in order to avoid changing `samza_metric_reporter.py`